addons/web_editor/static/src/xml/commands.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web_editor.banner">
+        <div t-attf-class="o_editor_banner o_not_editable lh-1 d-flex align-items-center alert alert-#{alertClass} pb-0" role="status" data-oe-protected="true">
+            <i t-attf-class="oe_unremovable fs-4 fa #{iconClass} mb-3" t-att-aria-label="label"/>
+            <div class="o_editable o_editable_no_shadow w-100 ms-3 oe_unremovable" data-oe-protected="false">
```

Comment:
I'd remove both `oe_unremovable` usage here, because it should not be mandatory since those should be in a `contenteditable="false"` context and they should not be removed by `deleteBackward` or `deleteForward` processes after:
https://github.com/odoo/odoo/pull/128567

As for the reason, `oe_unremovable` nodes can trigger editor rollbacks if a mutation removes them directly (or in a deleteRange), so we would spare our future selves a headache by not using it (since this template is saved as is in the `htmlField` content, modifying its value would require an upgrade script).

I have a last battle to fight against `o_editable` which is also considered `unremovable` by the editor util, and I don't understand why (if a node has to have both concepts, it should have both `o_editable oe_unremovable` classes imo)
cc @dmo-odoo